my-app/src/components/WatchList/watchlist.delete.tsx

```diff
@@ -0,0 +1,33 @@
+import { dispatchLocalWatchlist } from "../../features/watchlistReducer/watchlistState";
+import Champion from "../../features/champions/types/champion";
+import notifySucces from "../Toast/succes.toast";
+import notifyError from "../Toast/error.toast";
```

Comment:
1. I know there's this convention in angular to use dots in different file names. I think in react world the preferred naming is camelCase or PascalCase.
2. Looks like there's no jsx in this file so no need to use `.tsx` as file extension

Reply:
Thanks, fixed in latest commit.